scripts/codemod/renameCmp.js

```diff
@@ -0,0 +1,89 @@
+// @flow strict
+
+/**
+ * CODEMOD to RENAME GESTALT COMPONENTS
+ * Ex. <Flyout /> to <Popover />
+ *
+ * TO RUN THIS CODEMOD
+ * yarn run:codemod renameCmp  <folder/file path> --previousCmpName=<value> --nextCmpName=<value>
+ * E.g. yarn run:codemod renameCmp  ~/code/pinboard/webapp --previousCmpName=Box --nextCmpName=RenamedBox
+ *
+ * OPTIONS:
+ * --previousCmpName: current component name to be replaced
+ * --nextCmpName: new component name to replace with
+ */
+
+import {
+  getImports,
+  getJSX,
+  getLocalImportedName,
+  initialize,
+  isNotGestaltImport,
+  matchesComponentName,
+  replaceImportedName,
+  replaceImportNodePath,
+  renameJSXElement,
+  saveSource,
+  sortImportedNames,
+  sourceHasChanges,
+} from './helpers/codemodHelpers.js';
+
+// $FlowFixMe[unclear-type]
+type GenericType = any;
+
+type FileType = { source: GenericType };
```

Comment:
ditto here…though also, these types seem to be shared and not likely to change, so should they be defined once in a more central location? we don't have to worry about docgen here so abstracting shared types is fine

Reply:
DONE